tests/add-to-cart.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { test } from '../pages/fixtures/homePage.fixture';
 import { expect } from '@playwright/test';
 
-test('should add product to cart', async ({ homePage }) => {
+test('@smoke Should add product to cart', async ({ homePage }) => {
```

Comment:
```suggestion
test('Should add product to cart', tag: ['@smoke'] ,async ({ homePage }) => {
```

Better way to use tags in playwright - https://playwright.dev/docs/test-annotations#tag-tests